remote-messaging/remote-messaging-impl/build.gradle

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2021 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'com.squareup.anvil'
+    id 'kotlin-kapt'
+}
+
+apply from: "$rootProject.projectDir/gradle/android-library.gradle"
+
+dependencies {
+    implementation project(path: ':di')
+    implementation project(path: ':common')
+    api project(path: ':remote-messaging-api')
+    api project(path: ':remote-messaging-store')
```

Comment:
why `api` and not `implementation`, specifically for the `remote-messaging-store`.

Do we need to recompile the `impl` module for every change in the `api` or the `store`?